src/main/java/ru/clevertec/userservice/security/model/request/RegisterJournalistRequest.java

```diff
@@ -0,0 +1,20 @@
+package ru.clevertec.userservice.security.model.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import ru.clevertec.userservice.user.model.request.CreateJournalistRequest;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class RegisterJournalistRequest extends CreateJournalistRequest {
```

Comment:
don't use Inheritance in response/request objects
do it only if you have BaseRequest/BaseResponse model for all models
but Register operation cannot extends Create operation in terms of OOP